lang/cs/Org.Apache.REEF.Common/TeleMetry/Counter.cs

```diff
@@ -0,0 +1,126 @@
+﻿// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+
+namespace Org.Apache.REEF.Common.Telemetry
+{
+    [DataContract]
+    public class Counter : ICounter
+    {
+        /// <summary>
+        /// Name of the counter.
+        /// </summary>
+        private readonly string _name;
+
+        /// <summary>
+        /// Description of the counter.
+        /// </summary>
+        private readonly string _description;
+
+        /// <summary>
+        /// Time that the counter is updated.
```

Comment:
What's the time unit used here? Seconds since the epoch? Please document.

Also, will these values be compared between machines? If so, this could be tricky to do.